README.md

```diff
@@ -13,13 +13,88 @@ If you missed IPFS Camp or want to revisit its content, everything will remain a
+| <img src="https://avatars2.githubusercontent.com/u/6493975?s=460&v=4" width="150" /> | Edgar Lee ([@hinshun](https://github.com/hinshun)) | Containerd meets IPFS | soon™ |
+| <img src="https://avatars1.githubusercontent.com/u/5222525?s=460&v=4" width="150" /> | Antoine Eddi ([@aeddi](https://github.com/aeddi)) | Implementing libp2p over Bluetooth Low Energy | soon™ |
+| <img src="https://avatars1.githubusercontent.com/u/21236?s=460&v=4" width="150" /> | Irakli Gozalishvili ([@gozala](https://github.com/gozala))  | Inter Planetary Data Feed (IPDF) | soon™ |
+| <img src="https://avatars1.githubusercontent.com/u/1398860?s=460&v=4" width="150" /> | Gonçalo Pestana ([@gpestana](https://github.com/gpestana)) | Privacy in P2P networks: DHTs and IPFS | soon™ |
+| <img src="https://avatars2.githubusercontent.com/u/81581?s=460&v=4" width="150" /> | Victor Grishchenko ([@gritzko](https://github.com/gritzko))  | An overview of RON | soon™ |
+| <img src="https://avatars3.githubusercontent.com/u/248257?s=460&v=4" width="150" /> | Rüdiger Klaehn ([@rklaehn](https://github.com/rklaehn))  | Actyx's experiences on building with IPFS | soon™ |
+| <img src="https://avatars3.githubusercontent.com/u/11004800?s=460&v=4" width="150" /> | Jonathan Budd ([@cesarosum](https://github.com/cesarosum)) | Peergos: E2E encrypted storage and social network on IPFS | soon™ |
+| <img src="https://avatars3.githubusercontent.com/u/1017236?s=460&v=4" width="150" /> | André Cruz ([@satazor](https://github.com/satazor)) | Introducing Nomios | soon™ |
+| <img src="https://avatars0.githubusercontent.com/u/17089485?s=460&v=4" width="150" /> | Alexandre Trottier ([@postables](https://github.com/postables)) | Why open source matters and how individuals can further the movement | soon™ |
+
+Note: topics relevant to package managers have a 📦 suffix.
+
+### [🧬 Sci-Fi Fair](SCI-FI_FAIR)
+
+| Project           | Presenter         | Description | Interview
+| -------------     | -------------     | ------------- | -- |
+| RPi cluster       | @hsanjuan, @lanzafame   | ipfs-cluster... on Raspberry Pi | soon™ |
+| Actyx             |  @rklaehn  | Industrial automation on IPFS | soon™ |
+| npm-in-a-box      | @achingbrain | what is says on the tin | soon™ |
+| The Gathering     | @JustMaier | conference networking on decentralized networks | soon™ |
+| Textile Photos    | @andrewxhill, @carsonfarmer, @balupton | a decentralized photo booth | soon™ |
+| Lazer Catbot      | @gorhgorh | 😼 | soon™ |
+| FAPRS             | @MarneeDear | IPFS over Ham Radio | soon™ |
+| Brave             | @yrliou, @darkdh, @emerick | native IPFS in Brave browser! | soon™ |
+| ENS               | @makoto | resolving ENS/IPFS with DNS | soon™ |
+| IPFS Spider       | @magnshen | crawling the IPFS network | soon™ |
+| Cluster Labs Horizon + more | @vasa-develop | viz for the RPi cluster + more | soon™ |
+| libp2p-ble-transport | @aeddi | IPFS over other kinds of radio! | soon™ |
+
+### [🧩 Unconf](UNCONF)
```

Comment:
@jessicaschilling do we have a record of which Unconf sessions happened and their respective notes?